doc/src/build/event_api.md

```diff
@@ -6,17 +6,7 @@ Sui [Full nodes](fullnode.md) support publish / subscribe using [JSON-RPC](json-
 
 The client provides an [event filter](#event-filters) to limit the scope of events. Sui returns a notification with the event data and subscription ID for each event that matches the filter.
 
-# Event types
-
-A Sui node emits the following types of events:
- * [Move event](#move-event)
- * [Publish event](#publish-event)
- * [Transfer object event](#transfer-object-event)
- * [Delete object event](#delete-object-event)
- * [New object event](#new-object-event)
- * [Epoch change event](#epoch-change-event)
-
-## Move event
+## Move events
 
 Move calls emit Move events. You can [define custom events](https://examples.sui.io/basics/events.html) in Move contracts.
```

Comment:
thought: "Move events are emitted when explicitly generated from a Move module", or something like that. I think from the current text, i's not clear whether all move calls will emit move events

Reply:
agree, and then also include the context how/why move modules explicitly generate events.